scripts/eslint:check

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+eslint "{src,test}/**/*.ts"
```

Comment:
What if `eslint` isn't installed locally? The `yarn eslint` prefix ensures that the correct version from `package.json` is used right?

Reply:
It's still the same with this method. You can only use yarn to run these scripts and yarn would use the correct version.